Formula/opentsdb.rb

```diff
@@ -16,30 +16,36 @@ class Opentsdb < Formula
     sha256 cellar: :any_skip_relocation, high_sierra: "5bcdc828069e124c16e1e6c8b2eb6732d0ef88533c27f60fcbb0bec369aca375"
   end
 
-  deprecate! date: "2020-11-13", because: :does_not_build
-
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "openjdk@8" => :build
+  depends_on "python@3.10" => :build
   depends_on "gnuplot"
   depends_on "hbase"
   depends_on "lzo"
-  depends_on "openjdk@8"
+  depends_on "openjdk@11"
 
   def install
-    system "./configure",
-           "--disable-silent-rules",
-           "--prefix=#{prefix}",
-           "--mandir=#{man}",
-           "--sysconfdir=#{etc}",
-           "--localstatedir=#{var}/opentsdb"
-    system "make"
-    bin.mkpath
-    (pkgshare/"static/gwt/opentsdb/images/ie6").mkpath
-    system "make", "install"
+    with_env(JAVA_HOME: Language::Java.java_home("1.8")) do
```

Comment:
Need to check if `JAVA_HOME` is actually detected in configure scripts. Mainly for Linux which may end up arbitrarily picking OpenJDK version. macOS will use `/usr/bin/java` which is `JAVA_HOME`-aware.

Reply:
I don't have a solution for this yet, but I've seen an identical failure for `drake`, `hexgui`, and `voldemort` on Linux as well, all of which also use `openjdk@8` as well.  For those 3 formulae, the test works locally but fails in CI, so there must be some issue with how we are setting `JAVA_HOME` for some `openjdk@8` bottles.  

I downloaded the Linux bottle and I can see that the value we set `JAVA_HOME` to is `@@HOMEBREW_JAVA@@`.  That is a placeholder that will be replaced when the bottle is poured, and I'm not entirely sure how it is determined.  But if it is not set to the right value then that would explain why the test fails.

Reply:
One common issue for Linux is that some formulae have scripts that run `java` from `PATH`, which only works with linked `openjdk` formula. The usual fix I use is the write env script with `PATH=$JAVA_HOME/bin:$PATH`.

There is another issue on Linux test where hbase commands are failing with `ERROR: KeeperErrorCode = ConnectionLoss for /hbase/master`. I haven't had chance to figure out why.